table/server/master/src/main/java/alluxio/master/table/CatalogProperty.java

```diff
@@ -26,6 +26,7 @@ private CatalogProperty(String name, String description, String defaultValue) {
     super(name, description, defaultValue);
   }
 
-  public static final CatalogProperty DB_TYPE =
-      new CatalogProperty("db.type", "The type of database. Possible values are: hive", "");
+  public static final CatalogProperty DB_IGNORE_TABLES =
+      new CatalogProperty("catalog.db.ignore.tables",
+          "The comma-separated list of table names to ignore from the UDB.", "");
```

Comment:
does this option affect deletion? should we have a comment about that?

Reply:
For now, it is only related to getting tables from udb via udb sync.